$OpenBSD$

add local include dir, but make sure it's after WRKSRC/include
add png to linkflags and remove rt

Index: sys/scons/SConscript.game
--- sys/scons/SConscript.game.orig
+++ sys/scons/SConscript.game
@@ -301,11 +301,11 @@ local_env = g_game_env.Clone()
 local_env.Prepend(CPPPATH = '#/game')
 
 local_env.Append( CPPDEFINES = [ 'GAME_DLL' ] )
-local_env.Append( CPPFLAGS = [ '-Wno-deprecated', '-Winvalid-pch' ] )
+local_env.Append( CPPFLAGS = [ '-Wno-deprecated', '-Winvalid-pch', '-Iinclude', '-I/usr/local/include' ] )
 
 # Link the runtime statically, link statically against libpng.a
 # the corresponding package libpng12-dev must be available on the system
-local_env.Append( LINKFLAGS = [ '-lrt' ] ) #, '-lpng'
+local_env.Append( LINKFLAGS = [ '-lpng' ] ) #, '-lpng'
 
 local_env.Append(LIBS = [
 	# Uncomment this if you're on a x64 system - there is no libpng.a on Ubuntu x64 for instance
